src/helpers/gcov.ts

```diff
@@ -0,0 +1,23 @@
+import glob from 'fast-glob'
+import { isProgramInstalled, runExternalProgram } from "./util"
+
+export function generateGcovCoverageFiles(projectRoot: string, include: string[] = [], ignore: string[] = [], gcovArgs = '',) {
+    if (!isProgramInstalled('gcov')) {
+        throw new Error('gcov is not installed, cannot process files')
+    }
+
+    const globstar = (pattern: string) => `**/${pattern}`
+    const gcovInclude = ['*.gcno', ...include].map(globstar)
+    const gcovIgnore = ['bower_components', 'node_modules', 'vendor', ...ignore].map(globstar)
```

Comment:
Would it be worth using the results from one of the blocklist functions ([manualBlocklist], [globBlocklist])?

[manualBlocklist]: https://github.com/codecov/uploader/blob/701c7236bd1ea4ec258c1047654993ae881e46f9/src/helpers/files.ts#L30
[globBlocklist]: https://github.com/codecov/uploader/blob/701c7236bd1ea4ec258c1047654993ae881e46f9/src/helpers/files.ts#L46

Reply:
I used the same 3 defaults as in bash, but I see all 3 are inside manualBlocklist so I will swap to that